client/src/LayoutComponents/StudentProfile/StudentProfile.js

```diff
@@ -1,12 +1,68 @@
-import React from "react";
+import React, { Component } from "react";
+import axios from "axios";
+import email from '../../images/contact-icons/email.svg'
+import github from '../../images/contact-icons/github.svg'
+import phone from '../../images/contact-icons/phone.svg'
+import house from '../../images/contact-icons/house.svg'
+import Project from '../../SharedComponents/ProjectView'
 import "./StudentProfile.css";
 
-const StudentProfile = () => {
-    return (
-        <div className="profile-container">
-            Student Profile
-        </div>
-    );
-}
 
+
+class StudentProfile extends Component {
+    state = {
+        studentInfo: null,
+        studentProjects: []
+    }
+    componentDidMount() {
+        const { id } = this.props;
+        axios
+            .get(`/api/students/${id}`)
+            .then((res) => {
+                const studentProjects = res.data.map(student => student.image)
+                this.setState({
+                    studentInfo: res.data[0],
+                    studentProjects
+                })
+            })
+            .catch((err) => console.log(err));
+    }
+
+    render() {
+        const { studentInfo, studentProjects } = this.state;
+        console.log(studentInfo)
+        const projects = studentProjects.map((project, i) => {
+            return <Project key={i} projectImg={project} />
+        }
+        )
+        return (
+            <div>
+                {!studentInfo ? (<div style={{ marginTop: '200px' }}>Loading...</div>)
```

Comment:
@Rahaf-96 Just to be consistent, you could give this div a className and them move the styling into the css file.